python/podio_class_generator.py

```diff
@@ -359,29 +393,38 @@ def _preprocess_datatype(self, name, definition):
     data = deepcopy(definition)
     data['class'] = DataType(name)
     data['includes_data'] = self._get_member_includes(definition["Members"])
-    data['is_pod'] = self._is_pod_type(definition["Members"])
+    data['includes_jl'] = {'constructor': self._get_member_includes(definition["Members"], julia=True),
+                           'struct': self._get_member_includes(definition["Members"], julia=True),
+                           'is_pod': self._is_pod_type(definition['Members'])}
```

Comment:
```suggestion
                           'struct': self._get_member_includes(definition["Members"], julia=True)}
    data['is_pod'] = self._is_pod_type(definition['Members'])
```
This should fix the c++ warnings you see, as `is_pod` needs to be available at the top level in the templates